UnicornEngine/video/source/Camera.cpp

```diff
@@ -18,6 +19,8 @@ Camera::Camera(const glm::vec3& position, const glm::vec3& direction) : m_aspect
                                                                       , m_znear(0.1f)
                                                                       , m_zfar(1000.0f)
                                                                       , m_dirtyView(false)
+                                                                      , m_fovLowerBound(44.0f)
```

Comment:
I don't think we need this settings now. If anyone will be needed, he can add setters of getters or smth.